README.md

```diff
@@ -132,9 +136,9 @@ The countMVS.py script produces the following as output:
 
 * A csv file (mvsCount.csv by default, this filename can be overriden by a command line switch) which contains a
 listing of:
-    * The MVS count for the deployment
-	* The Time period selected by the user for the last time seen for events from log sources to be considered in the
-    count
+   * The MVS count for the deployment
+	* The Time period selected by the user for the last time seen for events from log sources to be considered in the count
+	* If the windows workstation check was skipped or not. If it has been skipped windows workstations will need to be manually removed from the results to calculate the MVS count
```

Comment:
Small detail but tidier to specify Windows with capital W consistently in the README.

Reply:
good point thats them updated now ;)